Assignment4/Formil-b190422/task06.py

```diff
@@ -0,0 +1,87 @@
+g.parse(github_storage+"/rdf/example5.rdf", format="xml")
+
+"""Create a new class named Researcher"""
+
+ns = Namespace("http://somewhere#")
+g.add((ns.Researcher, RDF.type, RDFS.Class))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.1: Create a new class named "University"**
+
+"""
+print("TASK 6.1")
+ns = Namespace("http://somewhere#")
+g.add((ns.University, RDF.type, RDFS.Class))
+# Visualize the results
+print("Printeamos las clases que se encuentran en el grafo, entre ellas, la de universidad:")
+for s, p, o in g.triples((None,None,RDFS.Class)):
+  print(s,p,o)
+print("------------------------------------------------------------------------------------------------------------------------------------------")
+"""**TASK 6.2: Add "Researcher" as a subclass of "Person"**"""
+print("TASK 6.2")
+ns = Namespace("http://somewhere#")
+rdfs = Namespace("https://www.w3.org/TR/rdf-schema/#")
+g.add((ns.Researcher, rdfs.subClassOf, ns.Person))
+# Visualize the results
+print("Printeamos todas las tripletas relacionadas con Researcher")
+for s, p, o in g.triples((ns.Researcher,None,None)):
+  print(s,p,o)
+print("------------------------------------------------------------------------------------------------------------------------------------------")
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+print("TASK 6,3")
+ns = Namespace("http://somewhere#")
+rdfs = Namespace("https://www.w3.org/TR/rdf-schema/#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.JaneSmith,vcard.FN,Literal("Jane Smith")))
+g.add((ns.JaneSmith, RDF.type, ns.Researcher))
+# Visualize the results
+print("Printeamos los triples relacionados con Jane Smith, entre ellos el que indica que es Researcher")
+for s, p, o in g.triples((ns.JaneSmith,None,None)):
+  print(s,p,o)
+print("------------------------------------------------------------------------------------------------------------------------------------------")
+"""**TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**"""
+print("TASK 6.4")
+ns = Namespace("http://somewhere#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.JaneSmith,vcard.FN,Literal("Jane Smith")))
+g.add((ns.JaneSmith,vcard.Given,Literal("Jane")))
+g.add((ns.JaneSmith,vcard.Family,Literal("Smith")))
+# Visualize the results
+print("Printeamos los triples relacionados con Jane Smith")
+for s,p,o in g.triples((ns.JaneSmith, None, None)):
+  print(s,p,o)
+print("------------------------------------------------------------------------------------------------------------------------------------------")
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+print("Task 6.5")
+ns = Namespace("http://somewhere#")
+rdfs = Namespace("https://www.w3.org/TR/rdf-schema/#")
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+g.add((ns.UPM,RDF.type,ns.University))
+g.add((ns.JaneSmith,ns.WorksAt,ns.UPM))
```

Comment:
you have never added worksAt as a property. You should add it first to your graph